plugin/pkg/admission/limitranger/admission.go

```diff
@@ -557,10 +562,7 @@ func PodValidateLimitFunc(limitRange *corev1.LimitRange, pod *api.Pod) error {
 					}
 				}
 			}
-		}
-
-		// enforce pod limits on init containers
-		if limitType == corev1.LimitTypePod {
+		} else if limitType == corev1.LimitTypePod { // enforce pod limits on init containers
```

Comment:
I'm not seeing the `if` that this is hanging an `else` off of, this appears to belong to a `for`? Is this valid?

Reply:
[Line528 ](https://github.com/kubernetes/kubernetes/pull/103170/files#diff-4ee4beda9b01237cf20bb75af7e640f09a00b593440bd87c35642d1791ef70a8R528)

```
if limitType == corev1.LimitTypeContainer {

} else if limitType == corev1.LimitTypePod { // enforce pod limits on init containers

}
```
This is just a cleanup. The key change in this PR is the adding lines 246-250.